spec/features/admin/admin_can_create_new_genre_spec.rb

```diff
@@ -19,6 +19,20 @@
         expect(current_path).to eq(genres_path)
         expect(page).to have_content("Action")
       end
+
+      scenario "I should get a flash message if it fails to create a genre" do
+        admin = User.create!(username: "admin", password: "password", role: 1)
+        allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(admin)
+
+        visit genres_path
+
+        click_on 'Create Genre'
+        fill_in 'genre[name]', with: ""
+        click_on 'Create Genre'
```

Comment:
why did you have to click on 'Create Genre' twice here? you should have been able to remove lines 29 and 30, effectively just loading the page as an admin and clicking the 'create genre' button once in order to see the error message.